htdocs/main.php

```diff
@@ -295,18 +280,18 @@ function tplFromRequest($param)
 // capturing output so that smarty can render
 $tpl_data['console'] = htmlspecialchars(ob_get_contents());
 ob_end_clean();
-
 switch(isset($_REQUEST['format']) ? $_REQUEST['format'] : '') {
 case 'json':
     print $tpl_data['workspace'];
     break;
 default:
     $smarty = new Smarty_neurodb;
     $smarty->assign($tpl_data);
-    $smarty->display('main.tpl');
+    if (!$anonymous) {
+        $smarty->display('main.tpl');
```

Comment:
should we rename `main.tpl` to `private_layout.tpl` or simply `layout.tpl` for consistency/clarity?